about/index.md

```diff
@@ -29,23 +35,26 @@ STIX is a collaborative effort that is led by DHS but driven by an open communit
 
 ## Frequently Asked Questions
 
-#### Who is STIX for? What does STIX do for me?
+## Who is STIX for? What does STIX do for me?
 
-STIX is for anyone involved in defending networks or systems against cyber threats, including cyber defenders, cyber threat analysts, malware analysts, security tool vendors, security researchers, threat sharing communities, and more. STIX provides a common language for describing cyber threat information so it can be shared, stored, and otherwise used in a consistent manner that facilitates automation.
+STIX is for anyone involved in defending networks or systems against cyber threats, including cyber defenders, cyber threat analysts, malware analysts, security tool vendors, security researchers, threat sharing communities, and more. 
 
-#### How do I get it?
+
+## How do I get it?
 
 The STIX Language is available on the [STIX website](https://stix.mitre.org/language/).
 
 Bindings and related tools to help process and work with STIX are [open source on Github](https://github.com/STIXProject).
 
-#### Where can I find examples of STIX data? Are there any STIX repositories?
+## Where can I find examples of STIX data? Are there any STIX repositories?
+
+Certain STIX repositories [are available](/adopters) are available for free, while others are managed by paid services.
```

Comment:
I like linking to the adopters page, but what does managed by paid services mean? Why not just say "while others are paid services".